packages/entities/entities-plugins/src/components/PluginForm.cy.ts

```diff
@@ -0,0 +1,1273 @@
+      alias?: string
+      status?: number
+    }): void => {
+      const url = `${baseConfigKM.apiBaseUrl}/${baseConfigKM.workspace}/schemas/plugins/validate`
+
+      cy.intercept(
+        {
+          method: 'POST',
+          url,
+        },
+        {
+          statusCode: params?.status ?? 200,
+          body: params?.mockData ?? plugin1,
+        },
+      ).as(params?.alias ?? 'validatePlugin')
+    }
+
+    const interceptKMScopedEntity = (params: {
+      entityType: string
+      mockData?: object
+      alias?: string
+    }) => {
+      cy.intercept(
+        {
+          method: 'GET',
+          url: `${baseConfigKM.apiBaseUrl}/${baseConfigKM.workspace}/${params.entityType}/*`,
+        },
+        {
+          statusCode: 200,
+          body: params?.mockData ?? scopedService,
+        },
+      ).as(params?.alias ?? 'getScopedEntity')
+    }
+
+    const interceptKMOperatePlugin = (params: {
+      method: 'GET' | 'PATCH'
+      id: string
+      mockData?: object
+      alias?: string
+      status?: number
+      credential?: boolean
+      entityId?: string
+    }) => {
+      const url = params?.credential
+        ? `${baseConfigKM.apiBaseUrl}/${baseConfigKM.workspace}/consumers/${params.entityId}/acls/${params.id}`
+        : `${baseConfigKM.apiBaseUrl}/${baseConfigKM.workspace}/plugins/${params.id}`
+
+      cy.intercept(
+        {
+          method: params.method,
+          url,
+        },
+        {
+          statusCode: 200,
+          body: params.mockData ?? plugin1,
+        },
+      ).as(params?.alias ?? 'operatePlugin')
+    }
+
+    beforeEach(() => {
+      // Initialize a new router before each test
+      router = createRouter({
+        routes: [
+          { path: '/', name: 'home', component: { template: '<div>ListPage</div>' } },
+          { path: '/select-plugin', name: 'select-plugin', component: { template: '<div>SelectPage</div>' } },
+        ],
+        history: createMemoryHistory(),
+      })
+    })
+
+    it('should show create form - cors plugin', () => {
+      interceptKMSchema()
+
+      cy.mount(PluginForm, {
+        global: { components: { VueFormGenerator } },
+        props: {
+          config: baseConfigKM,
+          pluginType: 'cors',
+        },
+        router,
+      })
+
+      cy.wait('@getPluginSchema')
+      cy.get('.kong-ui-entities-plugin-form-container').should('be.visible')
+
+      // button state
+      cy.getTestId('form-submit').should('be.visible')
+      cy.getTestId('form-submit').should('be.enabled')
+      cy.getTestId('form-back').should('be.visible')
+      cy.getTestId('form-back').should('be.enabled')
+      cy.getTestId('form-cancel').should('be.visible')
+
+      // scope fields
+      cy.get('.field-selectionGroup').should('be.visible')
+      cy.get('.Global-check').should('be.visible')
+      cy.get('.Scoped-check').should('be.visible')
+      cy.get('.field-selectionGroup .field-AutoSuggest').should('not.be.visible')
```

Comment:
suggestion: you can use chaining for selectors like: cy.get('.field-selectionGroup').find('.field-AutoSuggest)